src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java

```diff
@@ -229,8 +225,9 @@ private UserDetails buildUserDetails(Submission submission) {
     }
 
     @Override
-    public ResponseEntity<FilingPackage> getSubmissionFilingPackage(UUID xAuthUserId, UUID id) {
-        Optional<Submission> fromCacheSubmission = this.submissionStore.get(id, xAuthUserId);
+    @RolesAllowed("efiling-user")
+    public ResponseEntity<FilingPackage> getSubmissionFilingPackage(UUID xAuthUserId, UUID submissionId) {
+        Optional<Submission> fromCacheSubmission = this.submissionStore.get(submissionId, xAuthUserId);
```

Comment:
Confirmation question: 
'xAuthUserId' doesn't need to be changed to 'xTransactionId'?

Reply:
yup yup nice catch